frontend/lib/scheduler/notification_scheduler.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:flutter_local_notifications/flutter_local_notifications.dart';
+import 'package:frontend/domain/task_entity.dart';
+import 'package:timezone/timezone.dart' as tz;
+
+class NotificationScheduler {
+  final FlutterLocalNotificationsPlugin _flutterLocalNotificationsPlugin = FlutterLocalNotificationsPlugin();
+
+  NotificationScheduler() {
+    _initializeNotifications();
+    _requestPermissions();
+    cancelAllNotifications();
+  }
+
+  Future<void> _initializeNotifications() async {
+    const AndroidInitializationSettings initializationSettingsAndroid = AndroidInitializationSettings('minmap/ic_launcher');  // TODO: Repair icon
```

Comment:
:information_source: I have problems with setting up icon for notification, I will repair it in the future